portfolio/src/main/webapp/style.css

```diff
@@ -201,24 +208,27 @@ h3 {
 .photo {
   position: relative;
 }
-/* Sets opacaity when hovered. */
+
 .photo:hover img {
+  /* Sets opacaity when hovered. */
   opacity: 0.3;
 }
+
 .photo:hover .overlay {
   opacity: 1;
 }
 
 /* Overlays a div with text, having absolute position with respect to its parent. */
 .overlay {
+  opacity: 0;
   position:absolute;
-  top: 50%;
   left: 50%;
+  top: 50%;
   transform: translate(-50%, -50%);
   -ms-transform: translate(-50%, -50%);
-  opacity: 0;
   transition: .5s ease-in-out;
```

Comment:
Follow alphabetical order here.

Reply:
For this part, `position` has to be before `top` and `left`, so that's as far as I can get to

Reply:
Actually never mind. I just tried and it seems as long as `position` is set `top` and `left` should be fine. The tutorial I saw was a bit misleading.